Helper/Entity/ProductHelper.php

```diff
@@ -1163,10 +1163,15 @@ public function canProductBeReindexed($product, $storeId, $isChildProduct = fals
         return true;
     }
 
+    /**
+     * @param Product $product
+     * @param int $storeId
```

Comment:
```suggestion
     * @param int $storeId
     
```
Sorry, I just checked the code '-_-

Reply:
@damcou, Fixed PHPdoc and added extra line between params and return, added method description.